lib/systems/default.nix

```diff
@@ -92,6 +131,8 @@ let
       isCompatible = _: throw "2022-05-23: isCompatible has been removed in favor of canExecute, refer to the 22.11 changelog for details";
       # Derived meta-data
       useLLVM = final.isFreeBSD || final.isOpenBSD;
+      useArocc = false;
+      useZig = false;
```

Comment:
Will this not override the input when these are set? (Might just be `final` being named confusingly…)